examples/experimental/Federated Learning with TrainConfig/run_websocket_server.py

```diff
@@ -0,0 +1,72 @@
+import argparse
+from multiprocessing import Process
+import logging
+
+import torch as th
+
+import syft as sy
+
+# from syft.frameworks.torch.tensors.interpreters import AutogradTensor
+from syft.workers import WebsocketServerWorker
+
+logger = logging.getLogger("run_websocket_server")
+FORMAT = "%(asctime)s %(levelname)s %(filename)s(l:%(lineno)d) - %(message)s"
+logging.basicConfig(format=FORMAT)
+logger.setLevel(level=logging.INFO)
+
+hook = sy.TorchHook(th)
+hook.local_worker = sy.VirtualWorker(id="local_worker_at_remote", is_client_worker=True, hook=hook)
+
+
+def start_proc(participant, kwargs):  # pragma: no cover
+    """ helper function for spinning up a websocket participant """
+
+    def target():
+        server = participant(**kwargs)
+
+        # data = AutogradTensor().on(th.tensor([[-1, 2.0], [0, 1.1], [-1, 2.1], [0, 1.2]], requires_grad=True))
+        # target = AutogradTensor().on(th.tensor([[1.0], [0.0], [1.0], [0.0]], requires_grad=True))
+        data = th.tensor([[-1, 2.0], [0, 1.1], [-1, 2.1], [0, 1.2]], requires_grad=True)
```

Comment:
We need to:

1. Clean up this file
2. Instantiate multiple remote workers
3. Split some real dataset (MNIST probably) between users

Reply:
This is contained in #2217